kss/builder/decanter/scss/_site-search.scss

```diff
@@ -0,0 +1,9 @@
+@charset 'UTF-8';
+
+// The site-search component
+
+#kssref-components-site-search {
+  .kss-modifier__example {
+    max-width: 40rem;
```

Comment:
That does look better in the styleguide. I wonder, tho, if it might mislead people into thinking that the site search has a max width by default, and therefore they don't have to specify a width for it? If we changed `.kss-modifier__example` to `.kss-modifier__wrapper`, then the example box would be smaller, and it might be more clear to users that the site search component does not specify a (max-)width.

Reply:
That's a good point. I can certainly do that.